pages/info/info.py

```diff
@@ -7,49 +7,151 @@
-
-            __Repeat Contributor__
-                > Definition for item here.
-
-            __Staleness__
-                > Definition for item here.
-
-            *__Note:__ These definitions are specifically for the data that is being used to populate this graph. It is not indictive of the
-            overall philosophy on open source communities.*
-
-            ### Page Community Health Lens
-            -------
-            Explaination of the grouping of visualizations on each page
-
-            ### Plotly Graph Functionality
-            -------
-
-            Plotly graphs have a mode bar if you hover over the top of the title.
-
-            If you want to reset the view of a graph with customization options, toggle one of the options to reset the view.
-
-
-        """
-        )
+        html.H3("Definitions"),
+        html.Hr(),
+        html.B("Contributor"),
+        html.Blockquote(
+            [
+                html.P(
+                    """
+                    An individual who has shared some software artifact or resource with an open source project.
+                    In practice, we are able to attach multiple emails to a given contributor identity, so
+                    a given contributor could make contributions under multiple emails.
+                    It is reasonable to say that an individual is a contributor if they have made one-or-more contributions
+                    to an open source project.
+                    """
+                ),
+            ]
+        ),
+        html.Hr(),
+        html.B("Contribution"),
+        html.Blockquote(
+            [
+                html.P(
+                    """
+                    A software artifact or resource that is shared with an open source project by some individual.
+                    There is no explicit, agreed-upon ranking among contributions. Where appropriate, we leave that
+                    to the discretion of the application user. There are, however, events that we do not consider
+                    contributions because nothing is shared with the project.
+                    """
+                ),
+                html.Br(),
+                html.H5("Examples of contributions:"),
+                html.Ul(
+                    [
+                        html.Li("Creating, reviewing, commenting on a Pull Request"),
+                        html.Li("Submitting, commenting on an Issue"),
+                        html.Li("Authoring, committing a Commit"),
+                    ]
+                ),
+                html.H5("Examples of non-contributions:"),
+                html.Ul(
+                    [
+                        html.Li("Starring a project"),
+                        html.Li("Forking a project"),
+                        html.Li("Watching a project"),
+                    ]
+                ),
+            ]
+        ),
+        html.Hr(),
+        html.B("Contributor Recency"),
```

Comment:
Not sure im following this one. Whats the inspiration for this? Not sure I see this as a definition

Reply:
Good note- we have three groupings of terms:[Fresh, Staling, Stale], [Active, Drifting, Gone], and [Fly-by, Repeat] and since they don't mix it makes sense for them to be defined together. 

"Contributor Recency" is a term I would have brought up to you if you hadn't mentioned it here- I don't immediately think of a better term for "how long has it been since someone has contributed" but lmk if you do.

Reply:
I dont have a different term for "contributor recency," for me that seems implied but dont mind including the definition. I do think its important to give context or definitions for the [Fresh, Staling, Stale], [Active, Drifting, Gone], and [Fly-by, Repeat] terms as we actively use them in our graphs

Reply:
My opinion is that the description below captures the definitions well, at least insofar as the three words that we're using [Active, Drifting, Away] have very little literal meaning. If you have a specific disagreement I'd love to know- I can change how something is said but I think the definition should be exclusively literal about the recency terms describing time ranges.